### PR TITLE
Code climate hints refactor

### PR DESCRIPTION
**Pull request by cezarpopa**

### CODE REVIEW

**Comment by markhowellsmead** [comment]
Thanks for the suggestions. I'll review them when you're finished (in January at the earliest).

**Comment by markhowellsmead** [comment]
Thanks for the suggestions. I'll review them when you're finished (in January at the earliest).

**Comment by cezarpopa** [comment]
Thank you!

**Comment by markhowellsmead** [comment]
I'm not sure of the purpose or necessity of this many changes to the code. This kind of rework would only usually be necessary if there is some functional reason to do so. The lack of any commentary regarding the purpose of so many changes (other than “tidying up” the code in line with an arbitrary system “Code climate”) means that I can't merge in these changes without a lot more information.

File: resources/Field.php [hunk 1]
```diff
@@ -47,27 +47,31 @@ public function __construct()
...
 			'label'        => __('Return Value', 'acf'),
 			'instructions' => __('Specify the returned value on front end', 'acf'),
 			'type'         => 'radio',
 			'name'         => 'return_format',
 			'layout'       => 'horizontal',
-			'choices'      => [
-				'post_object' => __('Form Object', ACF_GF_FIELD_TEXTDOMAIN),
-				'id'          => __('Form ID', ACF_GF_FIELD_TEXTDOMAIN)
-			],
+			'choices'      => $returnFormatChoices,
```
Comment (RupW) [inline resources/Field.php:61]: I've not used Code Climate and I'm curious about some of the things you've changed.

Was this a warning it raised? I think I prefer the original version. Is there a reason you shouldn't nest array initialisation like this?

File: resources/Field.php [hunk 2]
```diff
@@ -116,41 +116,51 @@ public function render_field($field)
 		// Create a css id for our field
 		$fieldId = str_replace(['[', ']'], ['-', ''], $field['name']);
 
+		$this->prepareDropdownMarkup($field, $fieldId);
+	}
+
+	protected function prepareDropdownMarkup($field, $fieldId)
```
Comment (RupW) [inline resources/Field.php:122]: Ditto why break this function? I have seen some static analysis tools warn about cyclomatic complexity > 10, but I make it 7 for the original function counting the open braces.

**Pull request closed without merging**